libs/component-sample/src/lib/feature/search-result-list-sample/search-result-list-sample.component.ts

```diff
@@ -2,21 +2,20 @@ import { Component, OnInit, ViewChild, AfterViewInit, ChangeDetectorRef } from '
 
 @Component({
   selector: 'search-result-list-sample',
-  templateUrl: 'search-result-list-sample.component.html'
+  templateUrl: 'search-result-list-sample.component.html',
 })
 
 export class SearchResultListSampleComponent {
 
   constructor(private change: ChangeDetectorRef) { }
 
-
-  items= [
+  items = [
     { text: 'First', id: 1 },
     { text: 'Second', id: 2 },
     { text: 'Third', id: 3 },
     { text: 'Fourth', id: 4 },
-    { text: 'Fifth', id: 5 }
+    { text: 'Fifth', id: 5 },
+    { text: 'Sixth', id: 6, isDualRecord: true },
```

Comment:
Do not add additional records, the dual display should simply add on data to existing items.